schema.sql

```diff
@@ -9,4 +9,23 @@ CREATE TABLE animals (
     weight_kg DOUBLE PRECISION
 );
 
-ALTER TABLE animals ADD COLUMN species VARCHAR(50);
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN species VARCHAR(50);
+
+CREATE TABLE owners (
+    id BIGSERIAL primary key, 
+    full_name varchar(255), 
+    age integer
+);
+
+CREATE TABLE species (
+    id BIGSERIAL primary key, 
+    name varchar(255)
+);
+
+ALTER TABLE animals DROP column species;
+ALTER TABLE animals ADD species_id INT;
+ALTER TABLE animals ADD FOREIGN key(species_id) REFERENCES species(id);
```

Comment:
- Optional
You can do the same in one line. ⬇️
`ALTER TABLE animals ADD species_id INT REFERENCES species(id);`